marshmallow/schema.py

```diff
@@ -353,6 +418,17 @@ def __repr__(self):
         )
 
     @property
+    def jit(self):
```

Comment:
This property would go on my proposed `Schema` subclass in the external library.